src/services/importTracker.ts

```diff
@@ -392,14 +392,14 @@ namespace ts.FindAllReferences {
             if (cancellationToken) cancellationToken.throwIfCancellationRequested();
             forEachImport(sourceFile, (importDecl, moduleSpecifier) => {
                 const moduleSymbol = checker.getSymbolAtLocation(moduleSpecifier);
-                if (moduleSymbol) {
-                    const id = getSymbolId(moduleSymbol).toString();
+                forEach(moduleSymbol?.declarations, declaration => {

```

Comment:
Spoke with @amcasey and we wonder if it makes more sense to grab the merged symbol first and do this (rather than walking all of the declarations and grabbing their unmerged symbols).

Reply:
@DanielRosenwasser It seems we can use a merged symbol in `convertExport` and not change `importTracker`.